legend-engine-xt-serviceStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/service/plugin/ServiceStoreExecutor.java

```diff
@@ -19,19 +19,32 @@
 
 public class ServiceStoreExecutor implements StoreExecutor
 {
-    static final ServiceStoreExecutor INSTANCE = new ServiceStoreExecutor(new ServiceStoreState());
-
     private final ServiceStoreState state;
+    private final ServiceStoreExecutionConfiguration serviceStoreExecutionConfiguration;
+
+    public ServiceStoreExecutor(ServiceStoreState state)
```

Comment:
should we get rid of this api?

Reply:
We build a default configuration with an empty credentialProviderProvider if no config is passed

Reply:
where is this used, I am just concerned folks would use this because its easy and face runtime issues. So if we can delete would prefer deleting so that team make a right choice